site/stylesheets/_c.flooper.scss

```diff
@@ -48,12 +48,33 @@
     background-color: #222;
     color: white;
   }
+
+  &[class*="--b"] {
```

Comment:
We could also have used image tags with `object-fit: contain`. We get points from beatles content for SEO. IE 10 [support?](https://github.com/bfred-it/object-fit-images)

- I'm totally in favor of class attribute selectors these days, but "with great power comes great responsibility" so we better add `[class*="c-flooper__block--b]`. Why? Because `<div class="a--bbey-road">` also matches, basically everything that has that string like `--b` contained in a word `--beatles`. We want to give good examples to the kids!